app/models/conference.rb

```diff
@@ -15,4 +17,10 @@ def tickets_left
     confirmed_attendances = attendances.select { |attendance| attendance.confirmed }
     tickets.to_i - confirmed_attendances.size
   end
+
+  private
+
+    def date_interval
+      errors.add(:end_date, 'must be later than start date') if starts_on > ends_on
```

Comment:
I would add the error to either `:base` or an existing attribute that's concerned with the validation; e.g. `starts_on` or `ends_on`.